apps/api/migrations/070_redesign_sessions_for_rotation.up.sql

```
@@ -0,0 +1,38 @@
+    id             UUID PRIMARY KEY DEFAULT gen_random_uuid(),
+    session_id     UUID NOT NULL REFERENCES sessions(id) ON DELETE CASCADE,
+    token_hash     TEXT NOT NULL UNIQUE,
+    parent_id      UUID REFERENCES refresh_tokens(id),
+    issued_at      TIMESTAMPTZ NOT NULL DEFAULT NOW(),
+    expires_at     TIMESTAMPTZ NOT NULL,
+    used_at        TIMESTAMPTZ,
+    revoked_at     TIMESTAMPTZ,
+    revoked_reason TEXT
+);
+
+CREATE INDEX idx_refresh_tokens_session_id ON refresh_tokens(session_id);
```

SQLFluff (4.2.2)

[error] CREATE INDEX should use CONCURRENTLY to avoid locking the table during the build.

(PG01)